src/app/components/card-page/card-page.component.scss

```diff
@@ -0,0 +1,199 @@
+@import 'colors/variables';
+
+.card-page-container {
+  .ngx-card {
+    margin-bottom: 20px;
+
+    .dot {
+      display: block;
+      width: 3px;
+      height: 3px;
+      border-radius: 50%;
+      background-color: $color-blue-grey-400;
+      margin: 0 5px;
+    }
+
+    .date-display {
+      color: $color-blue-grey-400;
+    }
+
+    .view-details {
+      display: flex;
+      color: $color-blue-400;
+      font-size: 15px;
+
+      i {
+        margin-top: 5px;
+        margin-right: 5px;
+      }
+    }
+
+    .card-gutter {
+      margin-left: 30px;
+    }
+
+    .card-action {
+      justify-content: space-between;
+      padding-right: 0 !important;
+      color: $color-blue-grey-350;
+
+      .ellipsis {
+        margin-right: 12px;
+        padding: 0;
```

Comment:
Not sure if this is causing it, but it seems the dropdown menu toggled from the ellipsis is slightly misaligned

![image](https://user-images.githubusercontent.com/62297014/99559274-37017480-298a-11eb-89c7-3942fee9cabc.png)

Reply:
I think someone mentioned issues with the dropdown caret not aligning with dropdown toggle before. @Hypercubed Do you remember?